packages/features/ee/common/components/LicenseRequired.tsx

```diff
@@ -62,20 +61,17 @@ const LicenseRequired = ({ children, as = "", ...rest }: LicenseRequiredProps) =
           Icon={AlertTriangle}
           headline={t("enterprise_license")}
           description={
-            <div
-              dangerouslySetInnerHTML={{
-                __html: DOMPurify.sanitize(
-                  t("enterprise_license_description", {
-                    consoleUrl: `<a href="${CONSOLE_URL}" target="_blank" rel="noopener noreferrer" class="underline">
-                ${APP_NAME}
-              </a>`,
-                    setupUrl: `<a href="${WEBAPP_URL}/auth/setup" class="underline">/auth/setup</a>`,
-                    supportMail: `<a href="mailto:${SUPPORT_MAIL_ADDRESS}" class="underline">
-                ${SUPPORT_MAIL_ADDRESS}</a>`,
-                  })
-                ),
-              }}
-            />
+            <Trans i18nKey="enterprise_license_description">
```

Comment:
Verified that the translation is being picked up 
![image](https://github.com/calcom/cal.com/assets/1780212/55cefa9a-f5e9-4a0e-8334-d625eec4964f)